src/Application/Controller/XMLRPC/Handler.php

```diff
@@ -254,6 +254,30 @@ public function getTicketInfo($ticket_id)
 			
 			return $this->_getTicketInfo($ticket);
 		}
+
+		/**
+		 * Reset given ticket to initial state.
+		 * Also reset the children of a meta ticket and
+		 * encoding tickets which depend on the given ticket.
+		 *
+		 * @param integer $ticket_id ticket identifier
+		 * @throws Exception
+		 * @return boolean
+		 */
+		public function resetTicket($ticket_id)
```

Comment:
Last remark, or more of a question: Do you think, now that you implemented this nice ticket type filter, it would be also to expose it via the API?
Like a resetTicket($ticket_id, array $ticket_types = []) method?